manifests/init.pp

```diff
@@ -56,6 +70,7 @@
   Optional[Integer] $urgency                         = undef,
   Optional[String[1]] $assignment_group              = undef,
   Optional[String[1]] $assigned_to                   = undef,
+  Servicenow_reporting_integration::ReportAttributes $incident_creation_report_attributes = ['failed_changes','corrective_changes'],
```

Comment:
What do you think of `incident_creation_report_attributes` => `incident_creation_conditions` ? I thought of the latter an hour or two ago and kind of like it b/c it's shorter, just as clear, and more generic (attributes are a subset of conditions). It's also a less wordy explanation to people ("you can specify your incident creation conditions using this param" vs. "you can specify the incident creation report attributes and the module will create incidents if at least one of the report's attributes is an incident creation attribute ...") which is almost always a good sign.

If you agree with my point, then I don't think the name-change should be too bad (find + replace) but if you want that in a follow-up PR then that's OK too. We should also make sure that change's captured elsewhere in the code (e.g. one other such change is `ReportAttributes` => `IncidentCreationConditions`).

Reply:
I don't mind it. I was never super happy with Attributes but ok with letting customers let us know if they were annoyed by it.

Reply:
Yup, it felt weird to me too. I think this is much better and clearer. Cool, thanks.

Reply:
Done.